swift/ql/lib/codeql/swift/frameworks/StandardLibrary/NsUrl.qll

```diff
@@ -0,0 +1,17 @@
+import swift
+private import codeql.swift.dataflow.DataFlow
+private import codeql.swift.dataflow.ExternalFlow
+private import codeql.swift.dataflow.FlowSteps
+
+/**
+ * A model for `NSURL` members that permit taint flow.
+ */
+private class NsUrlSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        ";NSURL;true;init(string:);(String);;Argument[0];ReturnValue;taint",
+        // TODO
```

Comment:
`NSURL` should be properly modeled in its own PR.

Reply:
And URL for that matter.

Reply:
Issue here: https://github.com/github/codeql-c-team/issues/1429